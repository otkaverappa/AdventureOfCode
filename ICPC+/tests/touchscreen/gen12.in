19
wldvxefchoxl 10
xjzlwunllvhb
fbvtaapxnsul
yznfpntoyszv
mmizkpapqxls
acpnxlaxmqfb
dyqbsryfeggq
xjhpiuiixzyw
lyvxqabtytxr
kxyqgegdpqty
kdgjegfrhbrx
dstcnsvxptqv 10
zwzroupzzxid
eqvltoiwibav
tvslqigshhjv
bawcxhgdzdos
tzqbctwyqqjj
zrbizmfcoefl
lnrkshelgwml
piliawrbnujo
irtwvyjgnash
iwsqsgbkqmsq
ilsyfdopuhlr 10
huzuvrbdpvtk
bxwrlqkwbcui
hixcsivzcvuz
oxegtaqwxmqk
eagfecplnmpf
wneakaayyggt
gwpfkhqojwvn
bkboyqvufzvp
axqadwtmtkrf
shwbgtrjesxe
kuzqtwhvvxvy 10
uqmpdeuvnsyv
mpesjdwwxvot
tvootmnnczcf
fyctrdqfuvxe
ytcypqrilhxg
tmtynxfvwkop
ngujbrpcmrab
htmvalbwzxwm
udjsnzjcgfmh
ybllvlmcgazj
nbfmydbugkpu 10
kyyqekzcolnj
ycnfenqsqveq
yfkeszacazug
ltizeykdcyki
laabxhtxoddg
fgihfcnswxuc
vehzctipviss
plrervmydvfk
zscxpwanbjmf
duxafptwbmcu
jpsmmzwlrzji 10
xlvaukgzgdal
uvkxhmrtdlfp
kcbddmmcxjdt
vlvcqxpksbib
oburncjzgkdj
wrlvcqqyblcr
ircctmejpacc
clekxhvvajqc
zjabueudwyfr
klccmgequkbt
rypthhwhqykn 10
deqbfxupjytx
eapaksuerlxy
uthluryxyqad
puuauoxaopby
jxecicdexkpr
epqehqhxmdxg
txhhoiixhmap
qdvoplfvcyzm
ogjbmihffqpv
yztfnvweassr
fznjxmvovgrh 10
pamwrdsrcnxq
ivukpobwpoip
dddykwfbwtxn
zpfbcermbnwt
baptqzitdnrp
jyshupvtfauj
gowhbuafwsan
riiwxaniafpw
vkpcmmnsclcd
gckewkroucmt
cabehtbcdsgs 10
gumifoontysp
llfhnubrweyd
xbidtoxakjiq
awftxzkklsta
mwuibslavtfo
kcqumbmmzshy
rtielehxabfb
vteqojgymzva
ahobzxarqkxc
oezphhsecwwq
ifrwemwgtmhv 10
jhozuldisdzb
msgqqeiyjbun
nsvjedgpmurg
hurzzqbnkjdb
nobxpymfsioz
nuozqfixczxb
papcjsfyggvy
gkdzsrafnogd
woaynzbfbqhk
loltugrdsuek
nercuahsoise 10
jujlksxxgirc
rkfjflwuppxm
peeeoxiyttlf
nkdttwxkgfvl
qtgijfuyjyey
xoyqjjwwtbro
xradwwrmrzwa
eqbnrhnqyljh
wfgsizgfqjkm
hdbyexziqcyj
jnzhzirxpxpz 10
yyephrbowemb
enjupjfbxeiy
pbvealezjkoq
dreaysbcgmyv
ydwvhhtwiqdl
bjkmvaeztkbr
detjqrgoudlc
mgawxfhyqtnm
trnncqefuyqk
rwbmboqnvsks
arsqnfegzrvb 10
jaidayqtutfy
hxlcqvusppic
unktggwqietk
eldbelzmikoa
ilszabbuqnqy
toodvknzvrca
cbmmocnwnhxn
kakanazjqpml
zanxtpxvrljh
nwfdecqocarp
csythngjntgi 10
jffcqolflqkr
sbhudyliqmda
zjjmfpwoubsm
pdtcvevnhdjk
buuugxufhdtm
upcqqwcibxkw
bglkluxoorix
pecyjwmdnovg
mzonxalaiwlt
qkiebsbszgqi
ecorsjzelpri 10
qeiyavuqfexj
wabxgsimwwgp
hfvsvncnrnnu
ihkqlhzkkajs
urftplixsdrn
sucmhqgrxsjl
cixmliehalap
wkorqggkcjyl
zeeyzoldwkpj
tvrtjtihezyu
ffgjofuplynk 10
mbnllewecnzl
jksnjrjqypzo
utditqsirivf
mrjqhjbqvufe
nqunhtbeoemj
wetqoovbigtr
rvjmroqggmtn
hurvafhwmaoa
rjdzqxqjszwl
porvbmkiiegi
jngvpvwgebfw 10
yxgsyefnsykv
kvgszmcibjgq
eezkfejgepac
tgqneajrxplw
dohgxnydtxny
cwginhkipcwv
fhocyabboihl
xfqqddqhcxpr
gbcxeyujhjnf
joixyrkxyanb
fgkjfbbnfdkj 10
dgumphtawbzw
sluqnjutpgew
khjpmvapcudt
bwuzxtyrfsks
bgntoupybzpo
wpfyjjsnfoof
hmyogkjkqydf
ssfutviqmnqy
ykleybliqjyy
uhimiltagzwb
wetiumgswtyx 10
xjhpvfnrpwdx
jzzrywvwcqey
clrbgraddhsy
ogrdewcpvejv
aereuwdzjucp
ncvqjpqzxhdb
ghtblczngqsc
ovdxrfpehmvs
bnszxxcdevgs
zfffwzkmunkn
