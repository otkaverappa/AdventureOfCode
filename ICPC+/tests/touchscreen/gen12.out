dyqbsryfeggq 39
lyvxqabtytxr 39
fbvtaapxnsul 42
kdgjegfrhbrx 46
acpnxlaxmqfb 48
xjzlwunllvhb 49
kxyqgegdpqty 52
yznfpntoyszv 59
xjhpiuiixzyw 68
mmizkpapqxls 75
eqvltoiwibav 37
zrbizmfcoefl 41
irtwvyjgnash 43
bawcxhgdzdos 44
tvslqigshhjv 48
iwsqsgbkqmsq 51
tzqbctwyqqjj 52
zwzroupzzxid 58
lnrkshelgwml 61
piliawrbnujo 61
eagfecplnmpf 33
huzuvrbdpvtk 42
shwbgtrjesxe 44
wneakaayyggt 44
axqadwtmtkrf 47
bxwrlqkwbcui 49
bkboyqvufzvp 51
hixcsivzcvuz 54
gwpfkhqojwvn 56
oxegtaqwxmqk 56
fyctrdqfuvxe 33
udjsnzjcgfmh 33
mpesjdwwxvot 37
uqmpdeuvnsyv 40
ytcypqrilhxg 42
tvootmnnczcf 43
htmvalbwzxwm 44
ngujbrpcmrab 45
tmtynxfvwkop 46
ybllvlmcgazj 48
plrervmydvfk 41
zscxpwanbjmf 41
fgihfcnswxuc 42
ltizeykdcyki 43
duxafptwbmcu 48
kyyqekzcolnj 48
vehzctipviss 48
yfkeszacazug 51
laabxhtxoddg 53
ycnfenqsqveq 53
klccmgequkbt 41
vlvcqxpksbib 45
ircctmejpacc 46
zjabueudwyfr 47
uvkxhmrtdlfp 51
wrlvcqqyblcr 54
xlvaukgzgdal 54
oburncjzgkdj 55
clekxhvvajqc 57
kcbddmmcxjdt 62
ogjbmihffqpv 39
yztfnvweassr 40
eapaksuerlxy 43
puuauoxaopby 46
qdvoplfvcyzm 46
jxecicdexkpr 47
deqbfxupjytx 48
uthluryxyqad 49
epqehqhxmdxg 53
txhhoiixhmap 54
gckewkroucmt 32
baptqzitdnrp 40
dddykwfbwtxn 41
jyshupvtfauj 41
zpfbcermbnwt 41
gowhbuafwsan 45
pamwrdsrcnxq 45
riiwxaniafpw 46
vkpcmmnsclcd 46
ivukpobwpoip 56
rtielehxabfb 32
llfhnubrweyd 34
vteqojgymzva 35
awftxzkklsta 38
kcqumbmmzshy 38
oezphhsecwwq 38
xbidtoxakjiq 40
mwuibslavtfo 44
ahobzxarqkxc 52
gumifoontysp 52
gkdzsrafnogd 33
msgqqeiyjbun 33
hurzzqbnkjdb 38
nsvjedgpmurg 40
jhozuldisdzb 41
papcjsfyggvy 41
loltugrdsuek 43
nobxpymfsioz 47
nuozqfixczxb 48
woaynzbfbqhk 52
jujlksxxgirc 36
peeeoxiyttlf 37
eqbnrhnqyljh 38
qtgijfuyjyey 39
wfgsizgfqjkm 39
xradwwrmrzwa 44
nkdttwxkgfvl 47
xoyqjjwwtbro 47
rkfjflwuppxm 49
hdbyexziqcyj 54
pbvealezjkoq 31
detjqrgoudlc 42
rwbmboqnvsks 42
ydwvhhtwiqdl 46
mgawxfhyqtnm 50
bjkmvaeztkbr 52
trnncqefuyqk 52
dreaysbcgmyv 53
enjupjfbxeiy 53
yyephrbowemb 55
nwfdecqocarp 42
kakanazjqpml 43
ilszabbuqnqy 45
jaidayqtutfy 45
zanxtpxvrljh 45
toodvknzvrca 46
cbmmocnwnhxn 47
unktggwqietk 47
eldbelzmikoa 59
hxlcqvusppic 60
buuugxufhdtm 30
pdtcvevnhdjk 34
qkiebsbszgqi 38
zjjmfpwoubsm 39
pecyjwmdnovg 41
sbhudyliqmda 41
bglkluxoorix 44
jffcqolflqkr 47
mzonxalaiwlt 47
upcqqwcibxkw 56
wkorqggkcjyl 37
qeiyavuqfexj 40
hfvsvncnrnnu 45
zeeyzoldwkpj 48
ihkqlhzkkajs 50
tvrtjtihezyu 51
cixmliehalap 56
urftplixsdrn 56
wabxgsimwwgp 57
sucmhqgrxsjl 61
nqunhtbeoemj 35
mbnllewecnzl 40
utditqsirivf 40
jksnjrjqypzo 41
porvbmkiiegi 41
rvjmroqggmtn 41
wetqoovbigtr 43
mrjqhjbqvufe 50
hurvafhwmaoa 52
rjdzqxqjszwl 55
yxgsyefnsykv 34
kvgszmcibjgq 36
gbcxeyujhjnf 39
dohgxnydtxny 42
joixyrkxyanb 44
fhocyabboihl 45
xfqqddqhcxpr 45
tgqneajrxplw 46
eezkfejgepac 49
cwginhkipcwv 53
bwuzxtyrfsks 33
wpfyjjsnfoof 37
ykleybliqjyy 37
bgntoupybzpo 38
hmyogkjkqydf 38
uhimiltagzwb 39
dgumphtawbzw 41
khjpmvapcudt 41
ssfutviqmnqy 43
sluqnjutpgew 49
xjhpvfnrpwdx 40
aereuwdzjucp 42
ncvqjpqzxhdb 42
clrbgraddhsy 44
ghtblczngqsc 44
bnszxxcdevgs 46
jzzrywvwcqey 47
zfffwzkmunkn 47
ogrdewcpvejv 50
ovdxrfpehmvs 50
